src/components/CharacterComponent.jsx

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import styled from "styled-components";
+
+import CharacterImg from "../shared/assets/fotoKim.jpeg";
+
+let scoreBar = "";
```

Comment:
waarom declare je deze variabele?

Reply:
ojee ik had nog niet mijn laatste opgeruimde versie gepushed

Reply:
Dit is nog oude code heb ik in mijn commit hierna al verwijderd